src/components/input-style/input-style.ts

```diff
@@ -95,7 +97,7 @@ export class MInputStyle extends ModulVue {
     }
 
     public get isLabelUp(): boolean {
-        return (this.hasValue || (this.isFocus && this.hasValue)) && this.hasLabel;
+        return (this.hasValue || (this.isFocus && this.hasValue)) && this.hasLabel || this.labelUp && this.hasLabel;
```

Comment:
Comme discuté, remplacer la ligne par :
`return (this.hasValue || (this.isFocus && this.hasValue) || this.labelUp ) && this.hasLabel;`